doc/new_spectrograph.rst

```diff
@@ -2,23 +2,95 @@
 New Spectrograph
 ****************
 
-Here are notes on how to build a new spectrograph
-from scratch or to add a new mode.
+Here are notes on how to add a new spectrograph from scratch or to add a new
+mode. To do so, you should install ``PypeIt`` following the development path;
+see :doc:`installing` and :doc:`dev/development`.
 
 Entirely New
 ============
 
+``PypeIt`` defines instrument-specific behavior/parameters using a python
```

Comment:
Wow, super-duper thanks for this!

Reply:
@tepickering -- Can you take a look at this as you just went through it?
And suggested edits.  Thanks!

Reply:
i gave it a quick scan and it looks good. i can give it a deeper look later today or over the weekend. there were no obvious omissions, at least.